doc/cla/corporate/jarsa.md

```diff
@@ -11,10 +11,6 @@ Alan Ramos  alan.ramos@jarsa.com.mx https://github.com/alan196
 List of contributors:
 
 Alan Ramos alan.ramos@jarsa.com.mx https://github.com/alan196
-Luis Triana luis.triana@jarsa.com.mx https://github.com/xluiisx
-Luis Triana luistriana.28@gmail.com https://github.com/xluiisx
-Oscar Garza oscar.garza@jarsa.com.mx https://github.com/ougc27
```

Comment:
This person has already made some commits, please do not remove it. If you want to specify they no longer are part of the company, you can write `(up to <date>)` with the date the change

Reply:
Thanks for your clarification.

This is done 👍